providers/flagd/README.md

```diff
@@ -130,3 +119,25 @@ found the flag is returned with reason `CACHED`.
 By default, the provider is configured to
 use [least recently used (lru)](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/LRUMap.html)
 caching with up to 1000 entries.
+
+### OpenTelemetry support
```

Comment:
This is only affect RPC?

Reply:
Uhh that's a good question but I'm not really sure.

I'm not sure what auto-instrumentation would be applicable in-process... cc @Kavindu-Dodan

Reply:
Yes. Right now this is only applicable for the RPC version. Trace for the gRPC sync stream don't make sense as we have long-running stream connections. And flag evaluation metrics can be easily collected from the metrics hook.

Reply:
I mention this is RPC only now.